src/components/pages/LoginPage/validations.tsx

```diff
@@ -0,0 +1,32 @@
+import { FormInputsType } from "./interface";
+
+const formInputsValidation = (values: {
```

Comment:
You could pass two translations needed for the error message. You are invoking this function in a component, where you are already using the `t` function, so I do with sth along those lines:

const formInputsValidation = (
  values: {
    email?: string;
    password?: string;
  },
  wrongEmailLabel: string

Reply:
Nice! Thank you :) I've added also `wrongPasswordLabel`, as we have two different messages :)

Reply:
Great! Although we don't want to have a` wrong passoword label`, but actually only `missing password label `, as discussed in context of validation. ;)